design/types.go

```diff
@@ -188,6 +190,9 @@ const (
 
 	// Any is the type for an arbitrary JSON value (interface{} in Go).
 	Any = Primitive(AnyKind)
+
+	// File is the type for a file.
```

Comment:
Maybe add a note explaining that this type can only be used in a multipart definition.

Reply:
I fixed it. https://github.com/goadesign/goa/pull/1612/commits/50a3a0d56661868b1ed798a1bcdcb80f2ea47850